generics/constants/common.js

```diff
@@ -50,5 +50,7 @@ module.exports = {
     "DISTRICT": "district",
     "SERVER_TIME_OUT" : 5000,
     "OK" : "OK",
-    "PROJECT" : "project"
+    "PROJECT" : "project",
+    "PROJECT_CERTIFICATE_RECIPIENT_TYPE" : "user",
```

Comment:
@VISHNUDAS-tunerlabs can we get user type  from userProfile ? change the key to DEFAULT_CERTIFICATE_RECIPIENT_TYPE